fdbclient/StorageServerInterface.h

```diff
@@ -108,6 +109,7 @@ struct StorageServerInterface {
 			if (Ar::isDeserializing) {
 				getKey = RequestStream<struct GetKeyRequest>(getValue.getEndpoint().getAdjustedEndpoint(1));
 				getKeyValues = RequestStream<struct GetKeyValuesRequest>(getValue.getEndpoint().getAdjustedEndpoint(2));
+				getKeyValuesAggregate = RequestStream<struct GetKeyValuesAggregateRequest>(getValue.getEndpoint().getAdjustedEndpoint(2));
```

Comment:
I think this is the problem: the getAdjustEndpoint index needs to be unique

Reply:
Thanks!